runtime/src/accounts_db.rs

```diff
@@ -1115,6 +1115,8 @@ pub struct AccountsDb {
     /// true if drop_callback is attached to the bank.
     is_bank_drop_callback_enabled: AtomicBool,
 
+    pub validator_initialized: Arc<AtomicBool>,
```

Comment:
From the perspective of runtime/, there's no such thing as a validator so I wonder if we can use an alternative name that's more relevant to this crate.  Maybe:
* `runtime_verified: Arc<AtomicBool>,`
* `runtime_verification_complete: Arc<AtomicBool>,`
* `startup_verification_complete: Arc<AtomicBool>,`

Reply:
I like `startup_verification_complete`. Replaced.